Terminal Commands.md

```diff
@@ -59,3 +59,12 @@ Some commands can be found in the `bin` directory.
 
 `-f` forcefully do a task i.e. without asking for confirmation<br/>
 `-r` recursively do a task(looks in subdirectories too)
+
+
+###### Vim commands:
+ 
+ vim <filename> : opens the file with name "filename" or creates the file with the name given as argument if the file does nt exist in the working directory.
+ 
+ i - allows editing the file.
```

Comment:
Remove this. This repo is not on how to use vim. Rather add commands for opening files in other commonly used programs like sublime text etc.